poetry.lock

```diff
@@ -2988,54 +2988,54 @@ files = [
 
 [[package]]
 name = "pyuwsgi"
-version = "2.0.23"
+version = "2.0.23.post0"
```

Comment:
A bit hesitant about this change as there doesn't seem to be much documentation around it. I *think* this is primarily to fix a compatibility issue with Python 3.12

Reply:
Yeah I forgot to do `--no-update` when I updated the lockfile. Fixed now